modular_skyrat/modules/vore/code/__HELPERS/vore_helpers.dm

```diff
@@ -0,0 +1,96 @@
+/proc/vore_replace(messages, mob/living/pred=null, mob/living/prey=null, belly=null)
+	if (!istext(messages) && !(islist(messages) && length(messages)))
+		return ""
+	var/message = istext(messages) ? messages : pick(messages)
+	var/list/replacements = list("%pred" = "[pred]", "%prey" = "[prey]", "%belly" = "[belly]")
+	for (var/replacement in replacements)
+		message = replacetext(message, replacement, replacements[replacement])
+	return message
+
+/proc/send_vore_message(atom/movable/pred, pred_message, prey_message, audience_message, pref_respecting, section=VORE_CHAT_TOGGLES, prey=null, replace=TRUE, ignored=null, audience=TRUE, only=null)
+	var/turf/T = get_turf(pred)
+	if (!pred || !pred_message || (!T && !only && audience))
+		return
+	var/list/hearers = only ? only : (audience ? get_hearers_in_view(DEFAULT_MESSAGE_RANGE, pred) : list(pred, prey))
+	for (var/mob/hearer in hearers)
+		if (!hearer || !hearer.check_vore_toggle(pref_respecting, section))
+			continue
+		if (ignored && (hearer in ignored))
+			continue
+		if (!only && audience && hearer.lighting_alpha > LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE && T.is_softly_lit() && !in_range(T,hearer))
+			continue
+		if (hearer == pred)
+			to_chat(hearer, pred_message)
+		else if (prey && hearer == prey)
```

Comment:
Im not sure if you really need the hearer == prey here if you already have prey.

Reply:
actually the hearer == prey is the part I do need, but I can get rid of the first part of that if statement